components/director/internal/domain/tenant/repository_test.go

```diff
@@ -1546,52 +1546,75 @@ func TestPgRepository_GetCustomerIDParentRecursively(t *testing.T) {
 
 func TestPgRepository_GetCustomerIDParentRecursivelyByExternalTenant(t *testing.T) {
```

Comment:
The test name is not adapted. Should be `TestPgRepository_GetParentRecursivelyByExternalTenant`